20201201 release note (#255)
thoth-station/core: RELEASES.md

# Thoth Release Notes

This release notes file describes new and important changes to the frontend services, toolbox, backend
services and the knowledge graph of Thoth.

Release cycles may vary from 2 to 4 weeks, depending on complexity of features or capacity of the team! For the current
state of planning, have a look at out [Projects](https://github.com/orgs/thoth-station/projects).

## Release 2020.12.01

After releasing some features to our backend and knowledge graph last time, we are now happy to announce the general 
availability of a new GitHub App called "Khebhut"! Please feel free to join our newly created developer chat.

For all the nitty gritty details of our features (backend or services) have a look at our
[Sprint Demo recordings](http://bit.ly/thoth-on-youtube).

### GitHub App: Khebhut (aka Kebechet)

After a rework on our backend, we are happy to announce that GitHub has approved our newly updated Junior Cyborg Developer
application "[Khebhut](https://github.com/thoth-station/kebechet)"! The main activity of Khebhut is to keep your dependencies

> up to data, create releases of your software (incl. bouncing version numbers). Please add Khebhut to your repository and

Reviewer: ```suggestion
up to date, create releases of your software (incl. bouncing version numbers). Please add Khebhut to your repository and
```

# Thoth Release Notes

> This release notes file describes new and important changes to the frontend services and toolbox and to backend

Reviewer: ```suggestion
This release notes file describes new and important changes to the frontend services, toolbox, backend
```

> services and the knowledge graph of Thoth.

Reviewer: ```suggestion
services, and the knowledge graph of Thoth.
```

Release cycles may vary from 2 to 4 weeks, depending on complexity of features or capacity of the team! For the current
state of planning, have a look at out [Projects](https://github.com/orgs/thoth-station/projects).

## Release 2020.12.01

> After releasing some features to our backend and knowledge graph last time, we are now happy to announce the avaiability

Reviewer: ```suggestion
After releasing some features to our backend and knowledge graph last time, we are now happy to announce the availability
```

> of new GitHub App to the general public! The two apps help you keeping your source code fresh and up to data, and

Reviewer: why are saying kebechet is new ? 
are we trying to say: `it is newly available in public space for helping the general public with dependent updates` 

and the `two apps`  is describe but we haven't spoken about any other app, is this regarding the `reproducible deployment pipeline via aicoe-ci`

apply Thoth's knowledge to your project. Please feel free to join our newly created developer chat.

For all the nitty gritty details of our features (backend or services) have a look at our
[Sprint Demo recordings](http://bit.ly/thoth-on-youtube).

### Developer Chat

We have created a public channel on Google Hangouts: [Thoth Station Developer Chat](https://chat.google.com/room/AAAAVjnVXFk),
this is meant as an open and interactive channel to ask questions with regards to Thoth's usage, development, deployment.
As we are a very Python focused community, feel free to ask how we (the cyborgs and the humans) can help!

### GitHub App: Khebhut (aka Kebechet)

> After a rework on our backend, we are happy to announce that GitHub has approved our new and updated Junior Cyborg Developer

Reviewer: the word new confuses me, should we change that to
```suggestion
After a rework on our backend, we are happy to announce that GitHub has approved our newly updated Junior Cyborg Developer
```
